datalad/distribution/create_sibling.py

```diff
@@ -102,7 +103,7 @@ def _create_dataset_sibling(
         # TODO: Is this condition valid for != '.' only?
         path_exists = True
         try:
-            out, err = ssh(["ls", remoteds_path])
+            out, err = ssh("ls {}".format(sh_quote(remoteds_path)))
```

Comment:
In any case, this implementation should work. I would not mind a better version with list support -- but I couldn't do it. At the same time I am suffering quite a bit from outdated systrem Git version. So I rather have it this way now, than some better way later.